cpplint/cpplint_unittest.py

```diff
@@ -3104,7 +3106,7 @@ def testBadCharacters(self):
     cpplint.ProcessFileData(
         'nul_utf8.cc', 'cc',
         ['// Copyright 2014 Your Company.',
-         unicode('\xe9x\0', 'utf8', 'replace'), ''],
+          six.ensure_text(b'\xe9x\0', encoding='utf8', errors='replace'), ''],
```

Comment:
indent is off -- should be aligned with [ above

also as above, you don't need `six`.  call `.decode()` on the byte directly.

Reply:
OK.